sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -4,30 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>Rule will always emit cloudwatch event even in case when rule is not sure about which training job to stop. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Added a statement about this